src/SprykerEco/Zed/Payone/Business/Payment/DataMapper/DiscountMapper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\DataMapper;
+
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer;
+
+class DiscountMapper
```

Comment:
* Add the interfaces to all mappers.
* The exception works only for the Persistence layer.

Reply:
Added